docs/ref/django-admin.txt

```diff
@@ -805,6 +805,20 @@ The behavior of this command changes depending on the arguments provided:
   migrated past the named migration. Use the name ``zero`` to unapply all
   migrations for an app.
 
+.. admonition:: ``django-admin migrate <app_label> zero``
+
+    The special zero ``<migration_name>`` to unapply(reverse) all the applied
+    migrations for the given ``<app_label>``.
```

Comment:
I think we should add a short section in `docs/topics/migrations/` instead of this admonition, e.g.
```
.. _migration-zero:

Reverting migrations
================

If you want to revert all migrations applied for an app ...
```

Reply:
Hi, just updated de PR. Hope it is clear.